packages/react-dom/src/events/__tests__/SyntheticClipboardEvent-test.js

```diff
@@ -9,24 +9,36 @@
 
 'use strict';
 
-var SyntheticClipboardEvent;
+var React;
+var ReactDOM;
 
 describe('SyntheticClipboardEvent', () => {
-  var createEvent;
+  var simulateEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticClipboardEvent = require('../SyntheticClipboardEvent').default;
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget').default(nativeEvent);
-      return SyntheticClipboardEvent.getPooled({}, '', nativeEvent, target);
+    React = require('react');
+    ReactDOM = require('react-dom');
+
+    document.body.innerHTML = '';
```

Comment:
Let's remove `container` from body at the end of each test instead? Or, better, declare variable once, and remove/add it before and after each test.